packages/odyssey-react/.storybook/data.js

```diff
@@ -0,0 +1,20 @@
+export const icons = [
```

Comment:
This will be used to render icons within the storybook. I left it because I was using it previously in the HTML Storybook and figured it would be useful at some point.

It _is_ cruft though, if you want it gone -- ill make it happen.

Reply:
You're right. When there is an Icon component to render, we'll use that.